Snippets/ServiceFabricPersistence/ServiceFabricPersistence_1/OutboxConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using NServiceBus;
+
+class OutboxConfiguration
+{
+    void OutboxConfigurationUsage(EndpointConfiguration endpointConfiguration)
+    {
+        #region ServiceFabricPersistenceOutboxConfiguration
+
+        var outboxSettings = endpointConfiguration.EnableOutbox();
+        outboxSettings.SetTimeToKeepDeduplicationData(TimeSpan.FromDays(1));
```

Comment:
Btw. I wonder if it was a good decision to put the extension methods into the NServiceBus root namespace. Since this persister only supports sagas and outbox isn't there a likelihood that another persister will be pulled in that has exactly the same extension methods in the root namespace and then it would clash? // cc @SimonCropp @Particular/nservicebus-maintainers